ddl/db_test.go

```diff
@@ -3449,8 +3449,13 @@ out:
 			c.Assert(err, IsNil)
 			break out
 		default:
-			tk.MustExec("update tnn set c2 = c2 + 1 where c1 = 99")
-			updateCnt++
+			// Close issue #14636
+			// Now the add column action has not been amendable, it will cause `Information schema is changed`
+			// error when it is finished in the process of a insert statement.
+			_, err := tk.Exec("update tnn set c2 = c2 + 1 where c1 = 99")
+			if err == nil {
```

Comment:
Should we check the content of this error?

Reply:
Not necessary, we only care about the final successfully inserted count, which is used for asserting later.